webapp/src/containers/Rhs/index.tsx

```diff
@@ -1,24 +1,49 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 
 import usePluginApi from 'hooks/usePluginApi';
 import {getprojectDetailsState, getRhsState} from 'selectors';
 
+import LinearLoader from 'components/loader/linear';
+
+import plugin_constants from 'plugin_constants';
+
+import AccountNotLinked from './accountNotLinked';
 import ProjectList from './projectList';
 import ProjectDetails from './projectDetails';
 
 const Rhs = (): JSX.Element => {
     const usePlugin = usePluginApi();
 
+    // Fetch the connected account details when RHS is opened
+    useEffect(() => {
+        if (getRhsState(usePlugin.state).isSidebarOpen) {
+            usePlugin.makeApiRequest(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName);
+        }
+    }, []);
+
     if (!getRhsState(usePlugin.state).isSidebarOpen) {
         return <></>;
     }
 
     return (
-        <div className='height-rhs bg-sidebar padding-25'>
+        <div className='overflow-auto height-rhs bg-sidebar padding-25'>
+            {
+                usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName).isLoading &&
+                <LinearLoader/>
+            }
+            {
+                usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName).isError &&
+                <AccountNotLinked/>
+            }
             {
-                getprojectDetailsState(usePlugin.state).projectID ?
-                    <ProjectDetails title={getprojectDetailsState(usePlugin.state).projectName}/> :
-                    <ProjectList/>
+                !usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName).isLoading &&
+                !usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName).isError &&
+                usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getUserDetails.apiServiceName).isSuccess && (
+                    getprojectDetailsState(usePlugin.state).projectID ?
+                        <ProjectDetails title={getprojectDetailsState(usePlugin.state).projectName}/> :
+                        <ProjectList/>
```

Comment:
Getting lint errors
![lint-errors-2](https://user-images.githubusercontent.com/92724755/185123012-481c4cd4-d6da-4761-8409-6c4b29bc6b39.png)

Reply:
this might be from your VS-code settings as on checking lint inside webapp not error/warning are produced